### PR TITLE
Feat/colors big calendar

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/components/BigCalendar/BigCalendar.tsx [hunk 1]
```diff
@@ -54,11 +65,13 @@ export function BigCalendar() {
...
               defaultDate={new Date()}
               defaultView="week"
+              eventPropGetter={eventStyleGetter}
               events={events}
               localizer={localizer}
-              onSelectEvent={(event: Event) => {
-                handleSelectEvent(event)
-              }}
+              onSelectEvent={handleSelectEvent}
+              // onSelectSlot={(slotInfo) => {
```
Comment (angelstackit) [inline src/components/BigCalendar/BigCalendar.tsx:72]: Please, remove commented code

File: src/layout/Header.tsx [hunk 2]
```diff
@@ -32,9 +34,9 @@ export function Header() {
     }
   }
 
-  if (typeof window !== 'undefined') {
-    if (window.location.pathname === '/login') return null
-  }
+  // if (typeof window !== 'undefined') {
+  //   if (window.location.pathname === '/login') return null
```
Comment (angelstackit) [inline src/layout/Header.tsx:38]: Please, remove comments from this TypeScript file.

File: src/components/BigCalendar/BigCalendar.tsx (hunk 1 above)
Comment (juliswer) [inline src/components/BigCalendar/BigCalendar.tsx:72]: Remove commented code

File: src/layout/Header.tsx [hunk 3]
```diff
@@ -32,9 +34,9 @@ export function Header() {
     }
   }
 
-  if (typeof window !== 'undefined') {
-    if (window.location.pathname === '/login') return null
-  }
+  // if (typeof window !== 'undefined') {
```
Comment (juliswer) [inline src/layout/Header.tsx:37]: Remove commented code

**Review by juliswer — APPROVED** [review]
Looks nice. Please check my comments before merging

**Pull request merged**